cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,71 @@
 /// <reference types='cypress' />
 
 describe('Web Tables page', () => {
-  it('', () => {
+  let user;
 
+  beforeEach(() => {
+    cy.visit('/');
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
+  });
+
+  it('Assert the pagination and rows count selection', () => {
```

Comment:
you do not check the pagination here. Need to click button and assert page changing. Also need to select rows count and assert that table contains exact rows amount